### PR TITLE
Add Deployment elements to Triage Docs

### PR DESCRIPTION
**Pull request by MushuEE**
Throw in some docs about how triage is updated/deployed.

/assign @michaelkolber

### CODE REVIEW

**Review by spiffxp — CHANGES_REQUESTED** [review]
Deployment is automated for us by prow, I would rather document that path than encourage use of the manual path.  If you feel it's necessary to have the manual info, I would put it behind a big scary warning about only being used for emergency purposes

File: triage/README.md [hunk 1]
```diff
@@ -191,3 +191,13 @@ See [Main Output](#main-output). This is only a subset of the main output.
 ## Updating JS dependencies for the web page
 
 See: `package.json` + `bazel run @yarn//:yarn install`
+
+# Deployment
+Triage runs as static HTML hosted in GCS that is updated as part of a [Prow Periodic](https://github.com/kubernetes/test-infra/blob/418c380ec2d6839cd87e16695c770552a7c2d505/config/jobs/kubernetes/test-infra/test-infra-periodics.yaml#L27).
+
+To update the triage image run `make push` from `./triage` which will trigger a [cloudbuild](https://cloud.google.com/cloud-build) using [`//images/builder`](https://github.com/kubernetes/test-infra/tree/master/images/builder). This will result in a fresh triage image within the cloud image registry of the `k8s-testimages` project. (See Container Registry -> Images)
+
+To update Triage frontend in Production or Staging run `make push-static` or `make push-staging` respectively.
```
[the remark this answers is not in the document]
Comment (MushuEE) [reply triage/README.md:200]: I really meant for manual, should I indicate that or not put any information in for manual updates?

**Comment by MushuEE** [comment]
#19150

**Comment by MushuEE** [comment]
My bad, messed up merge

**Pull request closed without merging**